runelite-client/src/main/java/net/runelite/client/plugins/implings/ImplingsConfig.java

```diff
@@ -43,11 +44,20 @@
 		NOTIFY
 	}
 
+	@ConfigSection(
+		name = "Impling Configuration",
```

Comment:
Maybe something more like this? (maybe "Impling Tier" makes more sense? idk)
```suggestion
		name = "Impling Type Settings",
```